docs/micro-rdk/servo/gpio.md

```diff
@@ -0,0 +1,133 @@
+}
+```
+
+{{% /tab %}}
+{{% tab name="Attributes example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pin": "16",
+  "board": "local"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-servo-name>",
+      "model": "gpio",
+      "type": "servo",
+      "namespace": "rdk",
+      "attributes": {
+        "pin": "<your-pin-number>",
+        "board": "<your-board-name>",
+        "frequency_hz": <int>,
+        "min_angle_deg": <int>,
+        "max_angle_deg": <int>,
+        "min_period_us": <int>,
+        "max_period_us": <int>,
+        "pwm_resolution": <int>
+      }
+    }
+  }
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "esp32",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": [16]
+      }
+    },
+    {
+      "name": "my_servo",
+      "model": "gpio",
+      "type": "servo",
+      "namespace": "rdk",
+      "attributes": {
+        "pin": "16",
+        "board": "local"
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `gpio` servos:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `pin` | string | **Required** | The GPIO number of the pin the servo's control wire is wired to on the board. |
+| `board` | string | **Required** | `name` of the [board](/micro-rdk/board/) the servo is wired to. |
+| `frequency_hz` | int | Optional | The frequency of [PMW](https://docs.arduino.cc/learn/microcontrollers/analog-output) pulses sent to the servo, in Hertz (*Hz*). <br> Default = [`300`] <br> Range = [`0`, `450`] |
+| `min_angle_deg` | int | Optional | The minimum angle in degrees that the servo can reach. <br> Default = `0` <br> Range = [`0`, `180`] |
+| `max_angle_deg` | int | Optional | The maximum angle in degrees that the servo can reach. <br> Default = `180` <br> Range = [`0`, `180`] |
+| `min_period_us` | int | Optional | Override the safe minimum [PMW](https://docs.arduino.cc/learn/microcontrollers/analog-output) pulse width in microseconds. <br> Default: `500` |
+| `max_period_us` | int | Optional | Override the safe maximum [PMW](https://docs.arduino.cc/learn/microcontrollers/analog-output) pulse width in microseconds. <br> Default: `2500` |
+| `pwm_resolution` | int | Optional | The resolution of the [PMW](https://docs.arduino.cc/learn/microcontrollers/analog-output) driver (for example, the number of ticks for a full period). <br> Default: `0` |
+
+{{% alert title="Tip" color="tip" %}}
+
+Refer to your servo's data sheet for [pulse-width modulation (PWM)](https://docs.arduino.cc/learn/microcontrollers/analog-output), rotation, and wiring specifications.
+
```

Comment:
could we also include the note about gpio servos reserving a PWM channel and timer at the frequency at which you've configured the servo (kind of like what we have in gpio motor)?

Reply:
Done!